libbeat/tests/integration/framework.go

```diff
@@ -866,6 +866,20 @@ func GetESClient(t *testing.T, scheme string) *elasticsearch.Client {
 
 }
 
+// FileContains searches for the specified string in a file and returns the first matching line.
+// The method reads the file line by line, looking for the first occurrence of the match string.
+// If the match is found, it returns the entire line containing the match.
+// If no match is found, it returns an empty string.
+//
+// Parameters:
+//   - filename: Path to the file to search in
+//   - match: String to search for in the file
+//
+// Returns:
+//
+//	The first line containing the match string or an empty string if not found.
+//
+// The test will fail if the file cannot be opened or if an error occurs while reading.
```

Comment:
[NIT]
1 - it does not match the usual style of doc-string we use
2 - the _Returns_ is redundant as it's already stated in the beginning